src/control/server/server.go

```diff
@@ -109,6 +140,17 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 			break
 		}
 
+		// If we have multiple I/O instances with block devices, then we need to apportion
+		// the hugepage memory among the instances.
+		if len(cfg.Servers) > 1 && cfgHasBdev(cfg) {
+			srvCfg.Storage.Bdev.MemSize = hugePages.FreeMB() / min(len(cfg.Servers), maxIoServers)
```

Comment:
This is meant to help with the transition between the current model and the new model where all SPDK interaction happens in the privileged binary.

Reply:
To answer the question more directly... I think you're right that we should probably flag it as an error rather than ignoring the extra ioserver configuration. Will create a ticket for that. It's been tricky to be in-between two models (single vs multi-server) for this long.